### ISSUE TITLE
Implement `test model` subcommand

### ISSUE THREAD

**Issue by andrey-kuprianov** [informalsystems/atomkraft#66]
This is the follow-up issue for #27 (and PR #61), as well as for #65, which implement generation, execution, and reporting for single ITF traces. This issue depends on #54 and #55, which implement the necessary programmatic interface from the Model module.

The tasks of this issue are:
 - get the trace from the model via interface defined in #54 and #55
 - store the obtained trace in `traces` folder
 - do the same processing as implemented in #27 and #65, but using the trace obtained in the previous step
 - modify slightly the implementation in #61, and hook there the default trace obtained via #55. 

This is the simplest and preferable route for Atomkrat prototype. 

### Alternative route for Atomkraft MVP

The alternative route is to generate another kind of Pytest using `@mbt` decorator, and refer from it directly to the model being used. But this route is:
- more complicated, because it will require the modification of `@mbt` decorator in `modelator`, in order to adopt the decorator to the changed `modelator` API, with TOML configs. 
- will _not_ lead to reproducible Python scripts, as the Pytest test will depend on the model, and not on already generated trace.

Implementation of the alternative route will require a more deep integration with Modelator, and implementation of caching there, in order to achieve the speed and reproducibility of the first route. _When implemented_ though, this will be the main and preferable way of Atomkraft operation, because the users will not need to concern themselves with the intermediate phase (ITF traces), if they don't want to. It will be a direct route from a model and a test assertion, to the execution of multiple generated traces against the testnet.
